modules/network-api/src/main/java/org/apache/ignite/internal/network/TrackableNetworkMessageHandler.java

```diff
@@ -77,10 +75,9 @@ private static void maybeLogLongProcessing(NetworkMessage message, long startTim
 
         if (durationMillis > MESSAGING_PROCESSING_LOG_THRESHOLD_MILLIS) {
             LOG.warn(
-                    "Message handling has been too long [duration={}ms, message=[{}]]",
+                    "Message handling has been too long [duration={}ms, message={}]",
                     durationMillis,
-                    // Message may include sensitive data, however it seems useful to print full message content while testing.
-                    includeSensitive() ? message : message.getClass()
+                    LOG.isDebugEnabled() ? message : message.toStringForLightLogging()
```

Comment:
Maybe previous condition (`includeSensitive()`) is important and should be retained?

Reply:
Yep, I believe that we should retain includeSensitive. Otherwise we will log full message that may contain sensitive data.

Reply:
Fix it